forge-gui/res/cardsfolder/upcoming/mishras_research_desk.txt

```diff
@@ -3,7 +3,7 @@ ManaCost:1
 Types:Artifact
 A:AB$ Dig | Cost$ 1 T Sac<1/CARDNAME> | Defined$ You | DigNum$ 2 | ChangeNum$ All | DestinationZone$ Exile | RememberChanged$ True | SubAbility$ DBChoose | StackDescription$ SpellDescription | SpellDescription$ Exile the top two cards of your library, then choose one of them. You may play that card this turn.
```

Comment:
SpellDesc ≠ Oracle

Reply:
I think @tool4ever already fixed this so i'm just gonna remove this